src/dispersion.cpp

```diff
@@ -1,16 +1,23 @@
-#include "dispersion.h"
+#include <stdlib.h> // abs()
 
+#include "dispersion.h"
 #include "rng.h"
 
 double dispersion_sources::roll() const
 {
+    double sum = 0.0;
     double this_roll = 0.0;
+    for( const double &source : normal_sources ) {
+        sum += source;
```

Comment:
This is not correct, if you add two normal distributions, the mean of the resulting distribution is sum of the means of the addends (conviently 0), and the square of its standard deviation is equal to the sum of the squares of the addends standard deviations.

More importantly, there is no reason to aggregate them like this, just retrieve one sample from each distribution and add them, that is a much more obviously correct result.